lib/inject.rb

```diff
@@ -0,0 +1,42 @@
+class Array
+  def my_inject(symbol,*start)
+    # if block_given?
+    if start.empty?
+      case symbol
+      when :+
+        start = 0
+      when :-
+        start = 0
+      when :*
+        start = 1
+      when :/
+        start = 1
+      end
+    else
+      start = start[0]
+    end
+
+    case symbol
+      when :+
+        self.each do |x|
+        start += x
+      end
```

Comment:
Lint/BlockAlignment: `end` at 23, 6 is not aligned with `self.each do |x|` at 21, 8